lte/gateway/c/core/oai/tasks/mme_app/mme_events.cpp

```diff
@@ -106,3 +107,21 @@ int s1_setup_success_event(const char* enb_name, uint32_t enb_id) {
       event_value, S1_SETUP_SUCCESS, MME_STREAM_NAME,
       folly::to<std::string>(enb_id));
 }
+
+int s1_setup_failure_event(
```

Comment:
looks like `clang-format` check is failing on this file, can you run formatter on this?